Makefile

```diff
@@ -32,7 +32,8 @@ RUN := run
 
 # Variables to be overridden on the command line, e.g.
 #   make test PKG=./storage TESTFLAGS=--vmodule=multiraft=1
-PKG          := "./..."
+PKG          := ./...
+TAGS         := netgo
```

Comment:
If we're going to use netgo unconditionally, we should also add -tags and -installsuffix to build/devbase/godeps.sh so that we preinstall these versions when glock dependencies change. 

But I'm not sure if using netgo all the time is the right thing to do. It breaks things like `.local` hostnames on OSX; I think we should only use it when it's really necessary to avoid dynamically linking libc.